pagopa/io-app
# feat(IT Wallet): [SIW-1852] Hide discovery banner after close

## Short description
This PR hides the discovery banner in the messages screen for six months after the user closes it.

Note that only the closable banner in the messages screen stays hidden, the one in the wallet screen is always visible.

## List of changes proposed in this pull request
- Added `hideDiscoveryBannerUntilDate` key to the `preferences` reducer
- Changed the selector in `landingScreenBannerMap`

## How to test
- Close the discovery banner in the messages screen and reload the app: it should be visible anymore
- Ensure the discovery banner in the wallet screen is still visible
